metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -487,7 +487,11 @@ def get_views_for_dataset(
             BigqueryView(
                 name=table.table_name,
                 created=table.created,
-                last_altered=table.get("last_altered", table.created),
+                last_altered=datetime.fromtimestamp(
+                    table.get("last_altered") / 1000, tz=timezone.utc
+                )
+                if table.get("last_altered") is not None
+                else table.created,
```

Comment:
So based on https://cloud.google.com/bigquery/docs/information-schema-tables#schema `creation_time` is a TIMESTAMP just like `last_altered` (at least in the [PARTITIONS view](https://cloud.google.com/bigquery/docs/information-schema-partitions#schema)). How come we do datetime conversion for one but not the other? Am I missing something?

Reply:
Seems like it just works this way :/ guess it's a fun features of the `__TABLES__` table...